Bottom: e7a9a198982f62970fdf43ffd0c25b449b07c3e3
Top:    5ad74ec06f4e24676790572573af0952fb480a3f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-20 22:12:31 +0100

Refresh of use-trackball-to-allow-rotatio

---

diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index 38f0c13..06cbd42 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -2682,7 +2682,7 @@ ghid_parse_arguments (int *argc, char ***argv)
   /* setup GL-context */
   gport->glconfig = gdk_gl_config_new_by_mode (GDK_GL_MODE_RGBA    |
                                                GDK_GL_MODE_STENCIL |
-                                               GDK_GL_MODE_DEPTH   |
+//                                               GDK_GL_MODE_DEPTH   |
                                                GDK_GL_MODE_DOUBLE);
   if (!gport->glconfig) {
     printf("Could not setup GL-context!\n");
